Engine/AlgorithmManager.cs

```diff
@@ -955,7 +955,7 @@ private static void ProcessDelistedSymbols(IAlgorithm algorithm, List<Delisting>
                     }
                     else
                     {
-                        var message = option.GetPayOff(option.Underlying.Price) < 0
+                        var message = option.GetPayOff(option.Underlying.Price) > 0
```

Comment:
Whoa, was this the actual bug here?

Reply:
Not the real bug, only wrong message shown, I introduced it in my previous PR :)